NetKet/Machine/pyactivation.hpp

```diff
@@ -1,47 +0,0 @@
-// Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#ifndef NETKET_PYACTIVATION_HPP
```

Comment:
Why is this file removed? Have the activation functions been moved some place else?

Reply:
Yes, this was a leftover that should have been removed before. Activations had been already moved to be a type of Layer. They are defined in `pylayer` basically